pyiron_atomistics/sphinx/base.py

```diff
@@ -2151,6 +2153,13 @@ def from_hdf(self, hdf):
             self.old_version = True
             pass
 
+    def clear_output_for_restart(self):
+        """Clears the output for restarted job. Temporary fix."""
```

Comment:
```suggestion
        """(Re)set  to a clean state."""
        # This function is part of a temporary fix for restart jobs (see SphinxBase.restart).
        # It could also be used when existing jobs should be reparsed (e.g. after code fixes, updates, etc.)
```